.github/workflows/cypress-workflow.yml

```diff
@@ -3,10 +3,14 @@ on:
   pull_request:
     branches:
       - main
+      - 1.x
```

Comment:
Am I correct in assuming this means we don't want this to run on `1.3` and before?

Reply:
That's a good point. I based this configuration on the one used by the alerting dashboards workflows. We may want to reconfiguring both repos to check the `1.*` branches considering the new branching strategy. For now, would it make sense to merge this as-is, and discuss enabling the checks for a future PR?